FlightPlanProject/OtherModsInterface.cs

```diff
@@ -0,0 +1,170 @@
+
+
+using System.Reflection;
+using UnityEngine;
+using KSP.Sim.Maneuver;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using BepInEx;
+using BepInEx.Logging;
+using BepInEx.Bootstrap;
+using BepInEx.Logging;
+using HarmonyLib;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using MonoMod.Cil;
+// ReSharper disable UnusedType.Global
+// ReSharper disable UnusedMember.Global
+
+// CF : this direct dependency cause the dll to be needed during build
+// it is not really needed, we can easyly hardcode the mode names
+// during the introduction of K2D2 UI it cause me many trouble in naming
+using K2D2;
+using ManeuverNodeController;
+
+using FlightPlan.UI;
+
+namespace FlightPlan;
+
+public class OtherModsInterface
```

Comment:
All othor mods (k2D2 and MNC) are gather here to lighten Main class. I did that because i've got trouble with UI classes taht got the same name.